src/CheckoutSdk/Payments/PaymentResponseProcessing.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace Checkout.Payments
+{
+    public class PaymentResponseProcessing
```

Comment:
We will likely change the name of the type to be consistent with others but will raise a separate PR for this.